app/models/setting/discovered.rb

```diff
@@ -66,6 +67,10 @@ def self.discovery_hostname_fact_array
     from_array Setting['discovery_hostname']
   end
 
+  def self.discovery_auto_bond?
+    Foreman::Cast.to_bool(Setting['discovery_auto_bond'])
```

Comment:
Is this really needed? I believe the value should already be a boolean.
In addition, this is not used anywhere.

Reply:
I don't think so, just added it as it's being done the same way the line below :laughing: